src/app/fyle/dashboard/stats/stats.component.ts

```
@@ -224,6 +239,21 @@ export class StatsComponent implements OnInit {
     }
   }
 
+  goToTeamReportsPage(state: ReportStates): void {
+    if (state === ReportStates.UNAPPROVED_TEAM_REPORTS) {
+      state = ReportStates.APPROVER_PENDING;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)